Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1323,6 +1323,18 @@ - (void)_updateOverlayRegions:(const WebKit::LayerPropertiesMap&)changedLayerPro
 
     scrollingCoordinatorProxy->updateOverlayRegionLayerIDs(overlayRegionsIDs);
 }
+
+- (void)_updateOverlayRegionsForCustomContentView
+{
+    if (![self _scrollViewCanHaveOverlayRegions:_scrollView.get()]) {
+        [_scrollView _updateOverlayRegionsBehavior:false];
```

Comment:
Should use `NO` instead of `false`, since the method takes a `BOOL` parameter.

I think there are some existing cases that could be fixed as well.